controllers/numaresourcesoperator_controller_test.go

```diff
@@ -138,224 +141,242 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 				{MatchLabels: label1},
 				{MatchLabels: label2},
 			})
+			nroKey = client.ObjectKeyFromObject(nro)
```

Comment:
this is a cheap operation, can be done inline. On top of that, we don't care SO MUCH about performance in test code, so in general let's be a bit biased towards inlining (https://about.sourcegraph.com/go/advanced-testing-in-go/#Repeat-yourself)

Reply:
I was not concern about performance here. I just thought that as "nroKey" was a derived from "nro" only it could be a good idea to do as with "nro", create it once and use it, that said I have no strong opinion against create the object multiple times in the test where it is used.

Reply:
OK, then it is ok to create it once, and the explanation you just added should be added as a comment (comments which explain "why" are almost always good), something like (random example)
```
the key is bound to nro, it doesn't change and has the same lifetime, so it makes sense to create alongside it
```